src/components/passwordUpdate.js

```diff
@@ -51,13 +51,17 @@ const PasswordUpdate = () => {
               setMessage("An error occurred: " + response.statusText);
             }
           })
-          .catch((err) => setMessage(""));
+          .catch((err) => {
```

Comment:
Existing code above: no need for the if - if an error occurs, it will go to catch